src/wallet/wallet.h

```diff
@@ -413,7 +450,17 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     //  keystore implementation
     // Generate a new key
-    CPubKey GenerateNewKey();
+    CPubKey GenerateNewKey(uint32_t nAccountIndex, bool fInternal /*= false*/);
```

Comment:
Why the default `false` value for `fInternal` has been commented out?

Reply:
not sure why it was like that,anyways uncommented it